supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchConfiguration.scala

```diff
@@ -56,6 +55,11 @@ class AwsBatchConfiguration(val configurationDescriptor: BackendConfigurationDes
     case false =>
     PathBuilderFactory
   }
+
+  val dockerCredentials = BackendDockerConfiguration.build(configurationDescriptor.backendConfig).dockerCredentials
+  val dockerEncryptionKeyName: Option[String] = dockerCredentials flatMap { _.keyName }
```

Comment:
How are these variables used? I don't see any references in the code below.

Reply:
I tried to be consistent with GCP backend. Do you think I should remove them?

Reply:
I don't know-- I'm just trying to understand why they are there/how they are used. If you remove them, does something break?

Reply:
The GCP backend does define those variables, but they are then actually used PipelinesApiInitializationActor. I don't see that the Aws code uses these variables, so there is no need to define them (I don't see that there is an interface or class being extended that requires them).

Reply:
@henriqueribeiro to clarify, yes I do think you should remove the unused variables.

Reply:
@cahrens sorry for the delay, I was ooo yesterday. I agree with you and just pushed the changes :)